Lombiq.Tests.UI.Samples/Helpers/SetupHelpers.cs

```diff
@@ -61,9 +61,21 @@ public static async Task RunSetupAndSignInDirectlyAsync(UITestContext context, s
     public static async Task<Uri> RunAutoSetupAsync(UITestContext context)
     {
         await context.GoToHomePageAsync();
-        AssertSetupSuccessful(context);
+        await AssertSetupSuccessfulAsync(context);
         return context.GetCurrentUri();
     }
 
-    private static void AssertSetupSuccessful(UITestContext context) => context.Exists(By.Id("navbar"));
+    private static async Task AssertSetupSuccessfulAsync(UITestContext context)
+    {
+        try
+        {
+            context.Exists(By.Id("navbar"));
+        }
+        catch
+        {
+            // Before throwing a not-too-helpful NoSuchElementException, verify if the browser log has something to say.
+            await context.AssertLogsAsync();
```

Comment:
On further investigation I was wrong about this. Actually the problem is, that the setup doesn't log into file, just the console. Something is not right here. The OrchardCore.Cms.Web 1.3.0 doesn't behave like this so it's something on our end...

Reply:
I believe this used before (and tests failed due to exceptions during setup, even if nothing got broken visibly) so please investigate.

Reply:
I have just verified that this problem is present on the current dev branch as well. See demo of it happening on both normal execution and UI test below. So this is not related to the current issue, and I'd like to handle it in a separate issue to not hold back this one any more. Are you okay with that?

https://user-images.githubusercontent.com/4007293/168492718-36f9b391-e511-470c-8bf3-6e88d7a80f77.mp4

https://user-images.githubusercontent.com/4007293/168492720-4db5d864-91b3-4843-996f-e283afda4457.mp4

As a short term fix [I have made it look for validation errors](https://github.com/Lombiq/UI-Testing-Toolbox/pull/119/commits/9bcd690d5502575ab915ba65d657d225edb09221). This would've been helpful for the problem I had with failed DB setup during development and looks a bit more intelligible:
![image](https://user-images.githubusercontent.com/4007293/168492825-da9449d9-8ec1-4978-bc45-24d911646027.png)

Reply:
I see. This is not what I meant but rather, exceptions when the setup is already running (like module migrations failing) I remember being logged and failing tests before. I can't recall that we ever checked what happens in the scenario you're showing.

OK, but please open an OSOE issue and start working on this afterwards.

Reply:
The fix and new issue is good, I'm just leaving this convo unresolved so it can be linked from the new issue.